modules/aws-outposts-requirements-limitations.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="aws-outposts-requirements-limitations_{context}"]
+= AWS Outposts on {product-title} requirements and limitations
+
+By configuring your {product-title} cluster to accommodate the requirements and limitations of this platform, you can manage the resources on your Outpost similarly to those on a cloud-based AWS cluster.
+
+* To extend an {product-title} cluster on AWS into an Outpost, you must have installed the cluster into an existing VPC.
+
+* {product-title} clusters on AWS include the `gp3-csi` and `gp2-csi` storage classes.
+These classes correspond to Amazon Elastic Block Store (EBS) gp3 and gp2 volumes.
+{product-title} clusters use the `gp3-csi` storage class by default, but AWS Outposts does not support EBS gp3 volumes.
+
+* An Outpost has its own availability zone in an AWS region and has a dedicated subnet.
+Edge compute machines deployed into an Outpost must use the Outpost availability zone and subnet.
+
+* AWS Outposts does not support AWS Network Load Balancers or Classic Load Balancers.
+To manages Ingress objects for your edge compute resources, you must install the AWS Load Balancer Operator so that you can use AWS Application Load Balancers in the AWS Outposts environment.
+If your cluster contains both edge and cloud-based compute instances that share workloads, additional configuration is required.
+
+* To create a volume in the Outpost, the CSI driver requires the Outpost Amazon Resource Name (ARN).
+The driver uses the topology keys stored on the `CSINode` objects to determine the Outpost ARN.
+To ensure that the driver uses the correct topology values, you must set the volume binding mode to `WaitForConsumer` and avoid setting allowed topologies on any new storage classes that you create.
+
+* When you extend an AWS VPC cluster into an Outpost, you have two types of compute resources.
+The Outpost has edge compute nodes, while the VPC has cloud-based compute nodes.
+The public AWS Elastic Block volume cannot attach to Outpost edge compute nodes, and the Outpost volumes cannot attach to public compute nodes.
+The Outpost volume logic uses node affinity to avoid attaching to public compute nodes, but the public AWS Elastic Block volume logic does not prevent it from attaching to Outpost edge compute nodes.
```

Comment:
```suggestion
The public AWS Elastic Block volume cannot attach to Outposts edge compute nodes, and the Outposts volumes cannot attach to cloud-based compute nodes as well, the limitation also for CSI snapshot restore volumes.
```
The section is introducing the limitation, so we could omit the second one, the second is try to explain why the first limitation exist. The Outpost volume object(PV) uses node affinity to avoid attaching to public compute nodes, but the public AWS Elastic Block volume object does not have node affinity prevent it from attaching to Outpost edge compute nodes. So user should aware this if they want to migrate some apps from cloud-based to edge they needs to consider the persistent volumes need to manual backup and restore(by other external tools couldn't use the csi snapshot or the same pvc across different compute nodes).

Reply:
Thank you for clarifying this for me! I think the cloud-based to edge migration use case would be helpful to mention here. I will add that so the user can recognize why the limitation is important for them in that case.